src/nls/messages_en.json

```diff
@@ -79,6 +79,7 @@
     "dashboard.secretsModal.heading": "Create Secret",
     "dashboard.secretsModal.primaryBtnText": "Submit",
     "dashboard.secretsModal.secondaryBtnText": "Close",
+    "dashboard.serviceAccounts.error": "Error:",
```

Comment:
can we merge this with `dashboard.eventListeners.error` and rename appropriately?

Reply:
can i do this in a separate PR? there are a couple of instances of this.